src/kixi/event_fixer.clj

```diff
@@ -194,6 +226,31 @@
   ([acc x]
    (inc acc)))
 
+
+(defn throttler
+  "Slow things down"
+  [xf]
+  (let [a (atom 0)]
+    (fn
+      ([] (xf))
+      ([acc]
+       (xf acc))
+      ([acc x]
+       (when (= 1 (mod @a 2))
+         (println "Having a little sleep")
+         (Thread/sleep 1000))
```

Comment:
why do you need to sleep if the problem is `MAX_CONNECTION`s?

Reply:
We don't - this is a throttler we don't actually use now.  But handy to have about to plug into the transducer if we ever do so I left it.

Reply:
Is it worth moving into a gist then rather than being a distraction here?